### PR TITLE
Feature/add logging page

### PR DESCRIPTION
**Pull request by destrim**

### CODE REVIEW

File: lib/view/logging_view.dart [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  }
+
+  Container orText() {
+    return Container(
+      margin: const EdgeInsets.only(top: 25, bottom: 15),
+      child: const Text('LUB', style: textStyle),
+    );
+  }
+
+  Container idInput() {
```
Comment (Przemyslaw5) [inline lib/view/logging_view.dart]:
```suggestion
  Container idLoginInput() {
```

File: lib/util/prefs_util.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+class PrefsUtil {
+  static Future<String> getUserId() async {
+    final prefs = await SharedPreferences.getInstance();
+    return prefs.getString('user_id') ?? "";
```
Comment (Przemyslaw5) [inline lib/util/prefs_util.dart:6]: It is not better to initialize as NULL?

File: lib/view/logging_view.dart [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  }
+
+  SnackBar userNotExistsSnackBar(String error) {
+    return SnackBar(
+        behavior: SnackBarBehavior.floating,
+        content: Text(error)
+    );
+  }
+
+  SnackBar serverErrorSnackBar(String statusCode) {
```
Comment (Przemyslaw5) [inline lib/view/logging_view.dart:167]: What do you think to change snackbar color to red?

**Review by Przemyslaw5 — CHANGES_REQUESTED** [review]

File: lib/util/prefs_util.dart (hunk 2 above)
Comment (destrim) [reply lib/util/prefs_util.dart:6]: Method isNotEmpty is called on returned value

File: lib/view/logging_view.dart (hunk 3 above)
Comment (destrim) [reply lib/view/logging_view.dart:167]: Changed

**Review by Przemyslaw5 — APPROVED** [review]

**Pull request merged**